src/pages/SubscriptionService.tsx

```diff
@@ -59,6 +60,12 @@ type SubscriptionServiceState = {
   timestampLogo: number;
 };
 
+function isValidIPSubnet(value: string): CIDR {
+  return value.lastIndexOf("/") === -1
+    ? (`${value}/32` as CIDR)
+    : (value as CIDR);
+}
```

Comment:
Yes, you're right. Maybe I can rename it something like `mappingToSubnet` or `mapInputToSubnet`. I need to improve the logic behind this check, because it takes as valid input any string with any characters behind `/`. Hope to solve this well soon.